deployment/ansible/deploy.yml

```diff
@@ -24,6 +24,8 @@
   remote_user: epoch
   # Facts are already collected in the previous playbook
   gather_facts: no
+  max_fail_percentage: 25
+  serial: "25%"
```

Comment:
I see the end of this playbook is `bin/epoch start` that does not guarantee that node is ok. May I suggest ending playbook with more thorough check e.g. on a specific useful HTTP API being up and responding meaningfully (e.s. `/v1/top`)?

Reply:
Here you go: 3e0acc1